Index: src/main/java/baseline/Solution42.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/*\r\n *  UCF COP3330 Fall 2021 Assignment 4 Solutions\r\n *  Copyright 2021 Matthew Spiker\r\n */\r\n\r\npackage baseline;\r\n\r\npublic class Solution42 {\r\n\r\n}\r\n\r\n//class to manage employee information (parse, organize, and output)\r\nclass Employee {\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/baseline/Solution42.java b/src/main/java/baseline/Solution42.java
--- a/src/main/java/baseline/Solution42.java	
+++ b/src/main/java/baseline/Solution42.java	
@@ -6,10 +6,28 @@
 package baseline;
 
 public class Solution42 {
+    //main method
+    public static void main(String[] args){
+        //create class object emp to reference Employee class
+
+        //create a constant String fileName "data\\exercise42_input.txt" to reference the input file later
 
+        //print the header for the table that should be output
+
+        //call printEmployees to read and output the information from the input file into the table properly
+    }
 }
 
 //class to manage employee information (parse, organize, and output)
 class Employee {
+    //method to print the information of the employees to the console from a file, will take in a string fileName
+    public void printEmployees(String fileName){
+        //open a try statement to contain file not found exceptions
 
+            //create a file reader to read from fileName
+
+            //open a while loop to read through the file until the next line is blank
+                //create a string array of strings read from the current line, split into three strings by commas using split
+                //print the strings separately and spaced properly to the console
+    }
 }
Index: gradle/wrapper/gradle-wrapper.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle/wrapper/gradle-wrapper.properties b/gradle/wrapper/gradle-wrapper.properties
new file mode 100644
--- /dev/null	
+++ b/gradle/wrapper/gradle-wrapper.properties	
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
Index: settings.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settings.gradle b/settings.gradle
new file mode 100644
--- /dev/null	
+++ b/settings.gradle	
@@ -0,0 +1,2 @@
+rootProject.name = 'exercise42'
+
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
new file mode 100644
--- /dev/null	
+++ b/build.gradle	
@@ -0,0 +1,28 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+group 'org.example'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.2'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.2'
+}
+
+test {
+    useJUnitPlatform()
+}
+
+run {
+    standardInput = System.in
+}
+
+application {
+    mainClass = "baseline.Solution42"
+}
\ No newline at end of file
Index: .gradle/buildOutputCleanup/cache.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/.gradle/buildOutputCleanup/cache.properties b/.gradle/buildOutputCleanup/cache.properties
new file mode 100644
--- /dev/null	
+++ b/.gradle/buildOutputCleanup/cache.properties	
@@ -0,0 +1,2 @@
+#Sun Oct 10 18:53:32 EDT 2021
+gradle.version=7.1
diff --git a/.gradle/vcs-1/gc.properties b/.gradle/vcs-1/gc.properties
new file mode 100644
diff --git a/.gradle/7.1/gc.properties b/.gradle/7.1/gc.properties
new file mode 100644
diff --git a/.gradle/7.1/dependencies-accessors/gc.properties b/.gradle/7.1/dependencies-accessors/gc.properties
new file mode 100644
